release_operator_with_konflux.md

```diff
@@ -0,0 +1,112 @@
+# Release the operator (for OCP) with Konflux
+
+Releasing the operator is a 3 stage operation:
+* Build the images using Konflux's pipelines to your workspace environment
+* Release these images to the staging repository. The images are basically inspected based on predefined rules by Konflux and deposited to the staging repository upon success.
+* Release the FBC fragment to the RH catalog staging index using the pullspec of the images pushed to the staging registry.
+
+Production releases builds on top of the staging releases to do more or less the same, except that in this case it goes though a more detailed scrutity, ending up in the production registry. Past this step, it's the same FBC fragment generation using the image pullspec in production.
+
+## Staging release
+
+* Filter the latest snapshot. Keep in mind that we need to filter based on the bundle push event since that will most probably contain the nudged update from the controller. The following commands sorts all the snapshots for the helm-operator application by timestamp in ascending order and displays the name, integration tests success status and the merge PR number and remote branch used in the commit:
+```console
+oc get snapshots --sort-by .metadata.creationTimestamp -l pac.test.appstudio.openshift.io/event-type=push,pac.test.appstudio.openshift.io/original-prname=orchestrator-operator-bundle-on-push -ojsonpath='{range .items[*]}{@.metadata.name}{"\t"}{@.status.conditions[?(@.type=="AppStudioTestSucceeded")].status}{"\t"}{@.metadata.annotations.pac\.test\.appstudio\.openshift\.io/sha-title}{"\n"}{end}'
```

Comment:
how access is gained to this cluster?

Reply:
https://olm.operatorframework.io/docs/reference/file-based-catalogs/

Reply:
Done